dispatcher/backend/src/common/external.py

```diff
@@ -150,6 +149,10 @@ def get_statement():
     )
 
 
+class ExternalIpUpdater:
+    update_fn = update_wasabi_whitelist
```

Comment:
```suggestion
    update = update_wasabi_whitelist
```

Calling an `update_fn` function makes little sense.
Also please include a doctstring for this class